test/test_runner.cc

```diff
@@ -39,6 +41,20 @@ std::string findAndRemove(const std::regex& pattern, int& argc, char**& argv) {
   return return_value;
 }
 
+bool findAndRemoveExact(absl::string_view option, int& argc, char**& argv) {
```

Comment:
Should we just use TCLAP here? It seems like we are reimplementing a bunch org arg parsing for tests. Also, is there any reason to make this a real server option with documentation? Probably not but just throwing it out there.

Reply:
We could; actually I have that in a patch saved away if we want to go in that direction.

OTOH we could use the TCLAP library to do this parsing rather than do this manually; I just copied the loop from above. I don't know if we can replace the one above because it uses the `--name=value` syntax rather than `--name value`, so I was trying to use a parallel approach but I'm not wedded to it.

Reply:
actually there's a third option I considered, using `#ifdef  ENVOY_CONFIG_COVERAGE` and not making it a command-line option at all. WDYT?

Reply:
+1 for simple. :)

Reply:
(I think you would have to ifdef out that test which checks the log but that is probably fine)